cloudbuild.yaml

```diff
@@ -12,28 +12,17 @@ steps:
     env:
     - DOCKER_CLI_EXPERIMENTAL=enabled
     - VERSION=$_GIT_TAG
```

Comment:
Instead of exporting an unused VERSION env variable, can we use this to override the useless `REV` make variable, like this:
```
make push-multiarch 'REV=$_GIT_TAG'
```

Or do substitutions only work under `env`?

Reply:
Added changes in Makefile to override REV value with REV_CI value passed as env in cloudbuild 

> do substitutions only work under env? 

I am also not sure about how substitutions works. May be you can tag someone who can comment on this